department-of-veterans-affairs/bip-archetype-service
# Cmapi2 276

Suggested validation steps:
- run ./genarchetype.sh
- full-text search bip-archetype-service to ensure "origin" only appears in legitimate URLs/refs in README files etc
- run ./gensvc.sh
- add MyTest project to STS. Build the project. Run the project and check the swagger ui

The scripts got a lot longer than expected, but worth reading through all the comment lines in them to understand what-all is being done.
